modules/nw-enabling-multicast.adoc

```diff
@@ -17,10 +17,16 @@ endif::[]
 
 You can enable multicast between pods for your project.
 
+ifdef::openshift-origin,openshift-online,openshift-enterprise,partner-roks,openshift-webscale[]
```

Comment:
Consider simplifying this conditional. I think should accomplish the same thing with less duplication:

```
.Prerequisites
* Install the OpenShift CLI (`oc`).
* You must log in to the cluster with a user that has the `cluster-admin`
ifdef::openshift-rosa,openshift-dedicated[]
or the `dedicated-admin`
endif::[]
role.
```